README.md

```diff
@@ -1,21 +1,42 @@
 # Intel® Extension for DeepSpeed*
 Intel® Extension for DeepSpeed* is an extension that brings Intel GPU (XPU) support to DeepSpeed(https://github.com/Microsoft/DeepSpeed). It comes with the following components:
 1. DeepSpeed Accelerator Interface implementation
-2. DeepSpeed op builders implmentation for XPU
+2. DeepSpeed op builder implmentation for XPU
 3. DeepSpeed op builder kernel code
 
 DeepSpeed would automatically use Intel® Extension for DeepSpeed* when it is installed as a python package.   After installation, models ported for DeepSpeed Accelerator Interface that run on DeepSpeed could run on Intel GPU device.
 
-Usage:
-1. Install Intel® Extension for DeepSpeed*
+## Installation
+
+Optional: need to install [oneAPI Base Toolkit Packages](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html) when using Kernel Injection for inference
 
-`python setup.py install`
+`Intel® oneAPI DPC++/C++ Compiler` is required by OP builders to support the JIT build for Kernel Injection.
+
+1. Install Intel® Extension for DeepSpeed*
+```python
+pip install intel-extension-for-pytorch
```

Comment:
typo? It should be intel-extension-for-deepspeed

Reply:
> typo? It should be intel-extension-for-deepspeed

fixed